test/python/transpiler/test_commutative_inverse_cancellation.py

```diff
@@ -757,18 +757,18 @@ def test_no_cancellation_across_reset(self, matrix_based):
 
         self.assertEqual(circuit, new_circuit)
 
-    @data(False, True)
-    def test_no_cancellation_across_parameterized_gates(self, matrix_based):
+    def test_cancellation_across_parameterized_gates(self):
         """Test that parameterized gates prevent cancellation.
         This test should be modified when inverse and commutativity checking
         get improved to handle parameterized gates.
         """
+        theta = Parameter("theta")
         circuit = QuantumCircuit(1)
         circuit.rz(np.pi / 2, 0)
-        circuit.rz(Parameter("Theta"), 0)
+        circuit.rz(theta, 0)
         circuit.rz(-np.pi / 2, 0)
 
-        passmanager = PassManager(CommutativeInverseCancellation(matrix_based=matrix_based))
+        passmanager = PassManager(CommutativeInverseCancellation())
```

Comment:
So I get why you did this because a parameterized circuit can't use matrix based inverse cancellation. But I'm wondering how it passed before, and what this PR did to change that.

Reply:
Good catch, this should not have been changed! That was a leftover from when we added parameter support and I forgot to revert this for 1.2 🙂 Reverted in 91850b88c.